F21/joelanzi/_toLowerCase.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from unicodedata import normalize
+
+## To Lowercase Function
+def _to_lower(word, language):
```

Comment:
We should add some comments regarding the lowercase function, describing input and output parameters.